### PR TITLE
Split Mamba and Micromamba installation docs

### PR DESCRIPTION
**Pull request by jonashaag**

### CODE REVIEW

File: docs/source/micromamba-installation.rst [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+   brew install micromamba
+
+
+Mamba-org releases
+******************
+Install script
+^^^^^^^^^^^^^^
+
+If you are using macOS or Linux, there is a simple way of installing ``micromamba``.
```
Comment (AntoinePrv) [inline docs/source/micromamba-installation.rst:34]:
```suggestion
If you are using macOS ,Linux, or GitBash on Windows  there is a simple way of installing ``micromamba``.
```

File: docs/source/micromamba-installation.rst [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+Simply execute the installation script in your preferred shell.
+
+For Linux and macOS install with:
```
Comment (AntoinePrv) [inline docs/source/micromamba-installation.rst:37]:
```suggestion
For Linux, macOS, or GitBash on Windows install with:
```

File: docs/source/micromamba-installation.rst [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+.. We use ``bash <(curl ...)`` instead of ``curl .. | bash`` as the latter does not work with prompts
+
+.. code:: bash
+
+   "${SHELL}" <(curl -L micro.mamba.pm/install.sh)
+
```
Comment (AntoinePrv) [inline docs/source/micromamba-installation.rst:44]: Should we also add the `install.bat` and `install.ps1` scripts on micro.mamba.pm ?

**Review by AntoinePrv — APPROVED** [review]

File: docs/source/micromamba-installation.rst (hunk 3 above)
Comment (jonashaag) [reply docs/source/micromamba-installation.rst:44]: I don't know those. Do they exist? Can you add the correct invocation to this PR?

**Pull request merged**